### ISSUE TITLE
Removed from Chrome store

### ISSUE THREAD

**Issue by cdown** [diophung/close-chrome-tabs#3]
First of all thanks for this great extension which I've found invaluable over the years.

Just wanting to let you know that this was removed from the Chrome store and now Chrome states "This extension violates the Chrome Web Store policy" on the extensions page.

For now I'll just load the unpacked extension.

**Comment by jonnylangefeld** [comment]
I came here just right now to say the same thing. It only becomes evident how essential these little tools are when they stop working. So thanks for writing it!
I tried at least 10 other shortcut tools on the chrome web store but none worked with everything I'm using on a daily basis like this one does.
I hope we can get it back onto the chrome web store!

**Comment by diophung** [comment]
I have no idea why Chrome removed it, they just claim the privacy policy link is not working. This extension requires nothing from you to function FYI. 

So I resubmitted it to Chrome Store with an updated privacy policy - let's wait for their review and it will be published automatically once it passes review.

**Comment by jonnylangefeld** [comment]
Yeah it was funny, I looked at some other extensions trying to do similar things and they need all kind of access like site data. And this addon doesn't need anything and got banned 🙈 
Thank you for resubmitting 🙏🏼
